File: GammaRaySetlist.Model/Song.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace GammaRaySetlist.Model
{
    public class Song : Entity
    {
        public string Name { get; set; }
        public bool Listened { get; set; }

        public virtual Band Band { get; set; }
    }
}

File: GammaRaySetList.Web/js/models/song.js
var app = app || {};

app.Song = new Backbone.Model.extend({
    defaults: {
        name: '',
        listened: false
    },

    setName: function (name) {
        this.set('name', name);
    },

    toggle: function () {
        this.set('listened', !this.get('listened'));
    }
});